processor/filterprocessor/testdata/config_strict.yaml

```diff
@@ -0,0 +1,59 @@
+receivers:
+    examplereceiver:
+
+processors:
+    filter/empty:
+        metrics:
+            include:
+                match_type: strict
+    filter/include:
+        metrics:
+            # any names NOT matching filters are excluded from remainder of pipeline
+            include:
+                match_type: strict
+                metric_names:
+                    - hello_world
+                    - hello/world
+    filter/exclude:
+        metrics:
+            # any names matching filters are excluded from remainder of pipeline
+            exclude:
+                match_type: strict
+                metric_names:
+                    - hello_world
+                    - hello/world
+    filter/includeexclude:
+        metrics:
+            # if both include and exclude are specified, include filters are applied first
+            # the following configuration would only allow metrics named "hello/world" to pass through
+            include:
+                match_type: strict
+                metric_names:
+                    - hello_world
+                    - hello/world
+            exclude:
+                match_type: strict
+                metric_names:
+                    - hello_world
+    filter/strictwithconfig:
+        metrics:
+            include:
+                match_type: strict
+                strict:
```

Comment:
what is this? just an error to load?

Reply:
Oh, removed this test.

This is actually ok, golang yaml parser just ignores extraneous fields. But I removed the StrictConfig from the filterset configuration in a previous PR, so there's no need to have an extra test fora non-existent field here.